src/main/resources/db/changelog/init/15062021_add_composite_pk_location_discount_table.yml

```diff
@@ -0,0 +1,19 @@
+databaseChangeLog:
+  - changeSet:
```

Comment:
let's place this changeSet in the same file where we create location_discount table